agent-inject/agent/agent.go

```diff
@@ -357,7 +357,7 @@ func (a *Agent) Patch() ([]byte, error) {
 	// Add a volume for the token sink
 	a.Patches = append(a.Patches, addVolumes(
 		a.Pod.Spec.Volumes,
-		[]corev1.Volume{a.ContainerTokenVolume()},
+		a.ContainerTokenVolume(),
```

Comment:
I'm not sure if it matters, but I think this will add both the `home` and `home-init` volumes to a pod, even if either the init or sidecar is disabled.

Reply:
It will be available in the `volumes`, but not mounted.  We could make this smarter in the future, but for now it's really not a big deal AFAIK.  Thoughts?

Reply:
Added, was easier than I thought.